lib/ood_core/job/adapters/slurm.rb

```diff
@@ -617,6 +617,10 @@ def get_state(st)
             STATE_MAP.fetch(st, :undetermined)
           end
 
+          def get_gpu_count(gres)
+            gres.to_s.scan(/gpu:[^,]*(\d+)/).flatten.map(&:to_i).sum
+          end
```

Comment:
Sorry! :sweat_smile: Ruby's parlance is very declarative, so you don't see a lot of `get` or `set` prefixes.

```suggestion
          def gpus_from_gres(gres)
            gres.to_s.scan(/gpu:[^,]*(\d+)/).flatten.map(&:to_i).sum
          end
```